samples/sensor/lps22hb/sample.yaml

```diff
@@ -4,7 +4,8 @@ tests:
   sample.sensor.lps22hb:
     harness: console
     tags: sensors
-    depends_on: i2c lps22hb
+    pre-filter: subsys_dt_compat_enabled("i2c")
```

Comment:
I wonder what is the difference between this and the above change:
```
- depends_on: i2c lis2dh
+ pre-filter: subsys_dt_compat_enabled("i2c") and dt_compat_enabled("st,lis2dh")
```

Reply:
the above change probably need to be changed, i2c in pre-filter, and lis2dh in filter.